tests/openai/test_openai_autolog.py

```diff
@@ -35,6 +35,24 @@
 ]
 
 
+def _extract_property_from_chunks(variant: ChatChunkVariant | CompletionsChunkVariant):
```

Comment:
Q: do we need to return function? Can't we define `_extract_property_from_chunks(variant: ChatChunkVariant | CompletionsChunkVariant, chunks)`?

Reply:
We don’t need to return a function. Returning a function was just a matter of personal preference.